crates/node/core/src/args/database.rs

```diff
@@ -9,8 +9,13 @@ use clap::{
     Arg, Args, Command, Error,
 };
 use reth_db::{mdbx::MaxReadTransactionDuration, ClientVersion};
+use reth_node_types::ByteSize;
 use reth_storage_errors::db::LogLevel;
 
+fn parse_byte_size(s: &str) -> Result<ByteSize, String> {
```

Comment:
can you please move this below the args struct where we have the `LogLevelValueParser` defined?

Reply:
Done, thanks